Installation instruvtions don't work

```bash
12:44 pm jspc@egg.zero-internet.org.uk :: ~/projects/framework-keyboards $(master) npm install
npm ERR! install Couldn't read dependencies
npm ERR! Darwin 10.8.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.10.33
npm ERR! npm  v2.1.11
npm ERR! path /Users/jspc/projects/framework-keyboards/package.json
npm ERR! code ENOPACKAGEJSON
npm ERR! errno 34

npm ERR! package.json ENOENT, open '/Users/jspc/projects/framework-keyboards/package.json'
npm ERR! package.json This is most likely not a problem with npm itself.
npm ERR! package.json npm can't find a package.json file in your current directory.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/jspc/projects/framework-keyboards/npm-debug.log
```
The README should reflect this needs to be done in subdirectories.

This issue complements https://github.com/ShaunYearStrong/framework-keyboards/issues/2

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ A simple on screen keyboard made in a few different frameworks, just for fun.
 
 ## Installation
 
-Requires npm and bower, to install dependencies run:
+Requires npm and bower, to install dependencies, navigate to the subdirectory of the keyboard you want to run, then run:
 
 ```
 npm install

